src/plugins/vis_types/timeseries/public/application/editor_controller.tsx

```diff
@@ -30,8 +30,9 @@ export class EditorController implements IEditorController {
     private embeddableHandler: VisualizeEmbeddableContract
   ) {}
 
-  render({ timeRange, uiState, filters, query }: EditorRenderProps) {
+  async render({ timeRange, uiState, filters, query }: EditorRenderProps) {
     const I18nContext = getI18n().Context;
+    const defaultIndexPattern = (await getDataStart().dataViews.getDefault()) || undefined;
```

Comment:
nit: what if we move `defaultIndexPattern` into context instead of props drilling. It will help us a little bit in future to simplify code and load default index only from one place